docs/golden_metrics.md

```diff
@@ -47,7 +48,29 @@ memoryUsage:
 | select    |      Yes      |  | Provide the field and function you want to display in the metric. You must only provide one field, but you can do aggregations, sums, etc. Always name the fields to make it easier to read. e.g. `sum((provider.httpCodeElb4XXCount.Sum OR 0) + (provider.httpCodeElb5XXCount.Sum OR 0)) AS 'Errors'`|
 | from |   No    | `Metric` | Choose where your metric gathers the information from |
 | where |   No    | empty string | In the case you need a more granular WHERE clause added into the query you can use this field. e.g. `provider='Alb'` |
-| facet |   No    | `entity.name` | Which field to use when doing a FACET. This field will be used when displaying the metric for several entities |
+| facet |   No    | empty string | An extra facet by a specific field to be added to the default facet by entityName |
 | eventId |   No    | `entity.guid` | The event attribute used to filter the entity. We recommend to use the default `entity.guid` which is generated automatically as part of the entity synthesis. |
+| eventName |   No    | `entity.name` | The name of the field in the event that references the entity name. By default `entity.guid` which is generated automatically as part of the entity synthesis. |
 
+### Roll-up entities
 
+In the cases that the entity type can be ingested from different sources and you need to provide a different query implementation you can use `queries` instead of `query`
```

Comment:
```suggestion
In the cases that the entity type can be ingested from different sources and you need to provide a different query implementation for each, you can use `queries` instead of `query`
```